### PR TITLE
try to integrate wiki in home page

### PR DESCRIPTION
**Pull request by pmp-p**
sure not pretty as html, but need to start somewhere, that also will facilitate community editing.

### CODE REVIEW

File: wiki/pygbag/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+***
+
+# *Templates*
+## In this chapter you will learn about templates in pygbag
+So, you might be wondering that how will the players play your game if your window looks so small and the console takes up so much of place. To fix this, you can change the layout of the webpage using templates.
+
+### Using other templates
+So currently, pygbag uses the default template which u might wanna change, for that there are 2 ways you can try
+#### 1. Using the in-built templates
+For that u can use the following command<br>
```
Comment (pmp-p) [inline wiki/pygbag/README.md]: u you

File: wiki/pygbag/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+`pygbag --template window-template.tmpl folder_name`<br>
+this would now use the window-template which is prebuilt in pygbag and this template would clear everything except the window, from the webpage and align your game window to the top right corner.
+
+#### 2. Installing a template and customizing it
+For this u can go to the static folder in the [pygbag repo](https://github.com/pmp-p/pygbag/tree/main/static) and from there install one of the templates and then you can use the following command:<br>
```
Comment (pmp-p) [inline wiki/pygbag/README.md]: u you

File: wiki/pygbag/README.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+1. go inside your game folder
+2. go inside the build folder
+3. go inside the web folder
+4. then select the contents inside the folder
+5. zip those contents
+
+### Uploading to itch.io
+So to upload the file
+1. Create a new project
+2. Select the Kinf of project as HTML
```
Comment (pmp-p) [inline wiki/pygbag/README.md]: kinf kind

File: wiki/pygbag/README.md (hunk 1 above)
Comment (SaketKunjathur) [reply wiki/pygbag/README.md]: ^^"

**Review by pmp-p — COMMENTED** [review]
lgtm

**Pull request merged**